packages/dappmanager/src/modules/ethClient/apiUrl.ts

```diff
@@ -14,12 +14,14 @@ export function getEthExecClientApiUrl(dnpName: string, port = 8545): string {
    * domain = "bitcoin.dappnode", "other.public.dappnode"
    * ```
    */
-  const domain = getPrivateNetworkAlias({
-    dnpName: dnpName,
-    serviceName: dnpName
+  
+  const containerDomain = determineNetworkAlias({
```

Comment:
This does not look 100% reliable. What if you call this function with a dnp name that is a multiservice with no main service?